### ISSUE TITLE
Publish releases and Docker images for arm64

### ISSUE THREAD

**Issue by errorhandler** [moov-io/achgateway#159]
Currently releases and docker images are only for amd64. Building the Docker image manually on arm64 works perfectly fine but it would be great if the official releases build for arm64.

**Comment by adamdecaf** [comment]
Agreed. Right now we're blocked on Github offering ARM64 runners that can make these images without emulation. There is [an open issue](https://github.com/actions/runner-images/issues/5631) tracking this support, but it's been slow going. Once those runners are available I plan to release ARM64 images. 

Internally at Moov we're having developers use the amd64 images without much trouble. We had trouble when running 20+ images under emulation though.

**Comment by adamdecaf** [comment]
https://github.com/actions/runner-images/issues/9255

**Comment by Makennsky** [comment]
I'd like to work on this

**Comment by adamdecaf** [comment]
Go for it. I'd like to have the script/steps as simple as possible while publishing both images. Ideally easily testable as well.

**Comment by Makennsky** [comment]
Hi,

The ARM64 / multi-arch Docker support discussed here has been implemented in PR #307:
https://github.com/moov-io/achgateway/pull/307

Could a maintainer please review it when possible? I’m happy to make any changes if needed.
Thank you.

**Comment by adamdecaf** [comment]
ARM64 releases/images are being landed in https://github.com/moov-io/achgateway/pull/379 (reimplements #307 on current master with native ARM runners).
